tests/test_virtual_chassis.py

```diff
@@ -0,0 +1,71 @@
+import time
+import json
+import redis
+import pytest
+from swsscommon import swsscommon
+
+class TestVirtualChassis(object):
+    def check_chassis_sysport(self, dvs):
+        # check if instance is connected to a chassis db
+        cfgFile = "/usr/share/sonic/virtual_chassis/default_config.json"
+        (ret, out) = dvs.runcmd("sonic-cfggen -v  DEVICE_METADATA.localhost.chassis_db_address -j %s" % cfgFile)
+        verifyChassDb = True
+        if ret != 0 or out.strip() == "":
+            # instance has no connection to chassis db
+            verifyChassDb = False
+            print "verify vs without chassisDb"
+        (ret, out) = dvs.runcmd("sonic-db-cli CHASSIS_DB keys SYSTEM_PORT\"*\"")
+        if verifyChassDb:
+            assert len( out.split() ) == 96
```

Comment:
Where does the 96 come from?

Reply:
chassis-db.json has entries to account for 3 linecards where each linecard has 32 sysports.